docs/api-docs.md

```diff
@@ -2376,7 +2429,41 @@ URL: {{server_url}}/register
 
 
 
-### 4. Update user
+### 5. Update other user
+
+
+
+***Endpoint:***
+
+```bash
+Method: PUT
+Type: RAW
+URL: {{server_url}}/users/:id
+```
+
+
+
+***URL variables:***
+
+| Key | Value | Description |
+| --- | ------|-------------|
+| id | 18 |  |
+
+
+
+***Body:***
+
+```js        
+{
+    "username": "updated_fake_email@gmail.com",
+    "role": "admin",
```

Comment:
since this will be ignored should we leave it out? same for updateUser

Reply:
I will leave it out for `updateUser`. This endpoint is created for admin console to do any changes they want to any users (e.g. can upgrade a normal user to admin) .